botfront/imports/api/instances/instances.methods.js

```diff
@@ -26,10 +31,84 @@ export const createInstance = async (project) => {
         }
     } catch (e) {
-        throw new Error('Could not cªreate default instance', e);
+        throw new Error('Could not create default instance', e);
     }
 };
 
+const getConfig = (model) => {
+    const config = yaml.safeLoad(model.config);
+    if (!config.pipeline) {
+        throw new Meteor.Error('Please set a configuration');
+    }
+    config.pipeline.forEach((item) => {
+        if (item.name.includes('fuzzy_gazette')) {
+            if (model.training_data.fuzzy_gazette) {
+                // eslint-disable-next-line no-param-reassign
+                item.entities = model.training_data.fuzzy_gazette.map(({ value, mode, min_score }) => ({ name: value, mode, min_score }));
+            }
+        }
+    });
+
+    config.language = model.language;
+    const apiHost = GlobalSettings.findOne({ _id: 'SETTINGS' }).settings.private.bfApiHost;
+    if (model.logActivity && apiHost) {
+        config.pipeline.push({
+            name: 'components.botfront.activity_logger.ActivityLogger',
+            url: `${apiHost}/log-utterance`,
+        });
+    }
+    return yaml.dump(config);
+};
+
+const getTrainingDataInRasaFormat = (model, withSynonyms = true, intents = [], withGazette = true) => {
+    if (!model.training_data) {
+        throw Error('Property training_data of model argument is required');
+    }
+
+    function copyAndFilter(obj) {
+        const copy = JSON.parse(JSON.stringify(obj));
+        delete copy._id;
+        delete copy.mode;
+        delete copy.min_score;
+        return copy;
+    }
+
+    // Load examples
+    let common_examples = model.training_data.common_examples.map(e => ExampleUtils.stripBare(e, false));
+    if (intents.length > 0) {
+        // filter by intent if specified
+        common_examples = common_examples.filter(e => intents.indexOf(e.intent) >= 0);
+    }
+
+    const entity_synonyms = withSynonyms && model.training_data.entity_synonyms ? model.training_data.entity_synonyms.map(copyAndFilter) : [];
+    const fuzzy_gazette = withGazette && model.training_data.fuzzy_gazette ? model.training_data.fuzzy_gazette.map(copyAndFilter) : [];
+
+    return { rasa_nlu_data: { common_examples, entity_synonyms, fuzzy_gazette } };
+};
+
+const getStoriesAndDomain = (projectId) => {
+    const { policies } = yaml.safeLoad(
+        CorePolicies.findOne(
+            { projectId }, { policies: 1 },
+        ).policies,
+    );
+    const mappingTriggers = policies
+        .filter(policy => policy.name.includes('BotfrontMappingPolicy'))
+        .map(policy => policy.triggers.map(trigger => trigger.action))
+        .reduce((coll, curr) => coll.concat(curr), [])
+        .join('\n  - ');
+    const mappingStory = `## mapping story\n* mapping_intent\n  - action_botfront_mapping_follow_up\n  - ${mappingTriggers}`;
+    const storyGroups = StoryGroups.find(
+        { projectId },
+        { stories: 1 },
+    ).fetch().map(group => group.stories.join('\n'));
```

Comment:
This wont work as group.stories is an array of objects

Reply:
fixed